app/src/main/java/org/merakilearn/ui/onboarding/PartnerFragment.kt

```diff
@@ -43,6 +43,12 @@ class PartnerFragment : Fragment() {
 
                 }
 
+                OnBoardingViewEvents.ShowCourseSelectionScreen -> TODO()
```

Comment:
They weren't implemented before, so we need to ask and check if we have to implement this or not. As of now to avoid errors related to when adding an else branch.